stock_available_portal/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+Open **Settings** > **Inventory**.
```

Comment:
**This text must be in CONFIGURE.rst** 

Open **General Settings** > **Inventory**.

 Configure the following filters in **Stock available to promise** section:

*  **Portal Visible Products** defines which products will be visible in portal
* **Visible to Users** defines which users have access to products in portal